### ISSUE TITLE
Menu designs if size below x by x

### ISSUE THREAD

**Issue by tenten8401** [lumien231/Custom-Main-Menu#80]
There should be different menu presets that you can make if the resolution is below a number

Just an example:

mainmenu_800x600.json
(Below 800x600)

**Comment by lumien231** [comment]
Are the gui scale menus not sufficient for your purpose?

**Comment by tenten8401** [comment]
Not really... Download IndustrialBox on Technic and resize it a bit and you'll see what I'm talking about.
(keep gui scale on medium)
